packages/base-styles/_variables.scss

```diff
@@ -43,6 +43,7 @@ $button-size: 36px;
 $button-size-small: 24px;
 $header-height: 60px;
 $panel-header-height: $grid-unit-60;
+$nav-sidebar-width: 300px;
```

Comment:
**Note:**

I'd prefer to scope this variable to the site editor package (in a new _variables.scss in `src/edit_site`), but it looks like the prevailing convention is to add variables to this global base-styles stylesheet. If anyone knows otherwise, I'm happy to make changes.